### ISSUE TITLE
Move to Sass

### ISSUE THREAD

**Issue by dimitraz** [alezucca/alessia2017#13]
To avoid repetition and improve modularity I think it would be a good move for us to switch over to Sass moving forward

**Comment by marcotroisi** [comment]
I agree. To begin with,  we could just copy and paste the current css into a scss file,  which should work out of the box. Then gradually replace syntax and break things into multiple files.

On Tue, 3 Jan 2017 19:27 Dimitra Zuccarelli, <notifications@github.com<mailto:notifications@github.com>> wrote:

To avoid repetition and improve modularity I think it would be a good move for us to switch over to Sass moving forward
